recipes/rdkit/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "rdkit" %}
+{% set version = "Release_2017_09" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/rdkit/rdkit/archive/{{ version }}.tar.gz
+  sha256: f2a360f515968e79845e6257be9fee85809ce40705d5b383598de3747dbfbe6f
+  patches:
+    - rdpaths.patch
+    - rdconfig.patch  # [win]
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - cmake
+    - requests
+    - boost <=1.64
+    - python
+    - numpy 1.11.*
```

Comment:
Is using the older versions of numpy usual for conda-forge? If not, there's no reason from the rdkit perspective to not use `>=1.12` here

Reply:
So the advice is usually to build against the oldest numpy you can: https://conda-forge.org/docs/meta.html

The run requirement then checks to see that that they have a numpy version at least that new. We might as well support numpy back to 1.11, since it wouldn’t stop you from using a newer version

Reply:
Ah, ok, that makes sense.

Reply:
Correct. This is fine.